blaserunning-and-shame/README.md

```diff
@@ -30,15 +30,27 @@ Shame is a uniquely exciting part of blaseball's rules: #watchparty on the offic
 
 ## Methodology and results
 
-Over a sample of 982 game logs, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
+Our sample consists of 982 game logs where the last out is present.
+Logs start in the middle of Season 2, and a number of games have incomplete logs due to various issues.
+
+On that sample, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
 The code for this research can be found in [shame.py](./shame.py), and source data can be found in `blaseball-archive-iliana.s3.us-west-2.amazonaws.com` (accessible via an S3 client; log timestamps 1596003134151 through 1596605456865).
 
 In this sample, 97 games (9.9%) resulted in shame under their original rule sets.
-Under new rules, 76 games (7.8%) resulted in shame; 29 games that originally resulted in shame no longer did, and 8 games now ended in shame that originally did not.
+Under new rules, 76 games (7.8%) resulted in shame; 25 games that originally resulted in shame no longer did, and 4 games now ended in shame that originally did not.
```

Comment:
"Under Enhanced Shame,"

Reply:
also, I believe this should technically be "76 games (7.8%) resulted in shame; 25 games that originally resulted in shame no longer did so; and 4 games ended in shame that originally did not" (notice the second semicolon)